addons/website/static/src/js/website.editor.js

```diff
@@ -1097,7 +1097,8 @@ var EditorBar = Widget.extend({
                         placement: 'auto top',
                     })
                     // Force-show popovers so users will notice them.
-                    .popover('show');
+                    .popover('show')
+                    .attr("contentEditable", true);
```

Comment:
@Gorash suppose there is failure in integer field (Like., Integer out of range, invalid literal for int()), then it's not editable after popup show.